Wrappers/Python/cil/optimisation/operators/SymmetrisedGradientOperator.py

```diff
@@ -78,7 +88,7 @@ def __init__(self, domain_geometry, bnd_cond = 'Neumann', **kwargs):
         
     def direct(self, x, out=None):
         
-        '''Returns E(v)'''        
+        r'''Returns :math:`E(v) = 0.5 * ( \nabla v + (\nabla v)^{T} )` '''        
```

Comment:
```suggestion
        r'''Returns :math:`E(v) = 0.5 * ( \nabla v + (\nabla v)^{T} )` 
        
        Parameters:
        -------------
        
        x: BlockDataContainer
        out: BlockDataContainer, default None
            If out is not None the output of direct will be filled in out, otherwise a new object is instantiated and returned.
        '''        
```